### PR TITLE
Pull Request Template Updated

### PR DESCRIPTION
**Pull request by deepak2431**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+##### If you have added any new feature please add screenshot of the output, how it has been changed after addition of the new feature
+
+...
+
+
+
+
+
+## Checklist
```
Comment (rahulkumaran) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Please start off the PR Template with the checklist! Make the necessary changes

**Comment by rahulkumaran** [comment]
Also, try squashing your commits. 10 commits are a lot to be merged!

**Comment by deepak2431** [comment]
@rahulkumaran I would made all the changes as requested by you and I would squash the commit too. Actually my vacation has over now and I have to go back to college. And I would be on train on 8th and 9th December. Could you please give me some extra time. I would make the PR on 10th December or definitely on 11th December.

**Comment by rahulkumaran** [comment]
No probs. I added squashed them. Thanks for the PR anyway.

**Pull request merged**